### DISCUSSION TITLE
English Text After Story Credits

### DISCUSSION THREAD

**Discussion by Wandering-Heiho** [dackst/nayuta#14] [Show and tell]
Soon after finishing up the previous cutscene, I was able to get back to making progress regarding the third and last cutscene with text that I wanted to edit:  the After Story credits.

Aside from hiding some of the names and then recreating them to match what is used in the rest of the translation, there isn't much else changed.

![Name Change Example](https://user-images.githubusercontent.com/73907314/115735999-ec30c200-a358-11eb-8bd5-2db8f15b2907.jpg)

You can see how it currently looks here:
https://youtu.be/j0zTKOuA1_U

As a side note, I was wondering if you're fine if one additional piece of text was added after the General Producer at around 4:00 the referred to the translation projects that have been done. If you would prefer that a message like this was not included, or said something different, feel free to let me know.
![Extra Message Example](https://user-images.githubusercontent.com/73907314/115735109-382f3700-a358-11eb-81b4-ad7c7d7bcc3d.jpg)

**Comment by dackst** [comment]
Looks nice so far. I'm fine with the extra text you added as-is. I like how it's not too in your face or anything.

I actually recently noticed that these credits give some romanizations for surnames (El Ordin, Rem Ordin, di Quarius) that I haven't found in any [other official sources](https://github.com/dackst/nayuta/blob/master/notes.md#changes-from-official-sources). I still don't like "Ordin" because of how it is likely to be pronounced by English speakers, but I can't think of any reason at all _not_ to use the rest (beyond the fact that "i" and "ji" seem to be more typical romanizations of [イ](https://en.wikipedia.org/wiki/I_(kana)) and [ジ](https://en.wikipedia.org/wiki/Shi_(kana))). 

Anyway, I was planning on changing "Il", "Lem", and "Ji" to "El", "Rem", and "di". Hopefully that won't be too much to change here.

**Reply by Wandering-Heiho** [reply]
Changing those should not be too much of a problem, as I thankfully left all of the names in an editable format.
I'll make sure to change Il to El, Lem to Rem, and Ji to di.

**Reply by Wandering-Heiho** [reply]
Alright, I've made the name changes to the [video](https://youtu.be/Vqun6n8ZE_I).

Here is the [link](https://drive.google.com/file/d/1D4IPBFl_HX7KST_A_H4ETDcNUz96YSZk/view?usp=sharing) to the .pmf file since GitHub doesn't want me posting a roughly 34MB file.
